guides/plugins/apps/storefront/cookies-with-apps.md

```diff
@@ -95,6 +99,195 @@ A `group` element consists of three child elements to configure the cookie group
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/trunk/src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd">
+    <meta>
+        <name>MyApp</name>
+        <label>My App</label>
+        <version>1.0.0</version>
+        <author>Your Name</author>
+    </meta>
+
+    <cookies>
+        <!-- Marketing/Tracking cookies go to Marketing group -->
+        <group>
+            <snippet-name>cookie.groupMarketing</snippet-name>
+            <entries>
+                <cookie>
+                    <cookie>myapp_conversion_tracking</cookie>
+                    <snippet-name>myapp.cookie.conversionTracking</snippet-name>
+                    <snippet-description>myapp.cookie.conversionTrackingDescription</snippet-description>
+                    <value>1</value>
+                    <expiration>90</expiration>
+                </cookie>
+                <cookie>
+                    <cookie>myapp_ad_targeting</cookie>
+                    <snippet-name>myapp.cookie.adTargeting</snippet-name>
+                    <value>1</value>
+                    <expiration>365</expiration>
+                </cookie>
+            </entries>
+        </group>
+    </cookies>
+</manifest>
+```
+
+```xml [Comfort Features]
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/trunk/src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd">
+    <meta>
+        <name>MyApp</name>
+        <label>My App</label>
+        <version>1.0.0</version>
+        <author>Your Name</author>
+    </meta>
+
+    <cookies>
+        <!-- User preferences go to Comfort Features group -->
+        <group>
+            <snippet-name>cookie.groupComfortFeatures</snippet-name>
+            <entries>
+                <cookie>
+                    <cookie>myapp_user_preferences</cookie>
+                    <snippet-name>myapp.cookie.userPreferences</snippet-name>
+                    <snippet-description>myapp.cookie.userPreferencesDescription</snippet-description>
+                    <value>{}</value>
+                    <expiration>365</expiration>
+                </cookie>
+            </entries>
+        </group>
+    </cookies>
+</manifest>
+```
+
+```xml [Required]
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/trunk/src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd">
+    <meta>
+        <name>MyApp</name>
+        <label>My App</label>
+        <version>1.0.0</version>
+        <author>Your Name</author>
+    </meta>
+
+    <cookies>
+        <!-- Essential cookies go to Required group (rarely needed for apps) -->
+        <group>
+            <snippet-name>cookie.groupRequired</snippet-name>
```

Comment:
I don't think it is necessary to write an example for each group. the way should be clear with one example :sweat_smile: